handbook/customers.md

```diff
@@ -1,63 +1,63 @@
-# Customer engineering
+# Customers
 
 ## Customer success
 
 ### Scheduling meetings with customers
-To schedule an ad hoc meeting with a Fleet customer, use the ["Customer meeting" Calendly link](https://docs.google.com/document/d/1tE-NpNfw1icmU2MjYuBRib0VWBPVAdmq4NiCrpuI0F0/edit#heading=h.v47bs6uo0jpk).
+To schedule an ad hoc meeting (a video chat with multiple people or a non-scheduled meeting) with a Fleet customer, use the ["Customer meeting" Calendly link](https://docs.google.com/document/d/1tE-NpNfw1icmU2MjYuBRib0VWBPVAdmq4NiCrpuI0F0/edit#heading=h.v47bs6uo0jpk).
 
 ### Missed Zoom Meeting | Etiquette with Customers
 This is a tutorial on how to respond when someone from outside the company misses a call.
 
-#### 48 hours before a meeting
+#### 48 hours before a meeting 
 
 L1: Check the calendar invite to determine if the customer has accepted the invitation.
 
-L2: If the customer has not accepted the invitation please reach out through appropriate channels to confirm attendance (Ex: EAs, Email, Slack).
+L2: If the customer has not accepted the invitation, please reach out through appropriate channels to confirm attendance (ex: EAs, email, Slack). 
```

Comment:
@tgauda, in reference to "EAs":
For easy reading and translation please spell things out and avoid acronyms.